krm/0_foundation/Kptfile

```diff
@@ -0,0 +1,14 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: 0_foundation
+packageMetadata:
+  shortDescription: Anthos CI/CD foundation layer
+openAPI:
+  definitions:
+    io.k8s.cli.setters.namespace:
+      x-k8s-cli:
+        setter:
+          name: namespace
+          value: YOUR_NAMESPACE
```

Comment:
I don't think we're meant to provide a value upfront? Also this should be marked as required.

Reply:
Is there a way to create a kpt setter without a value? Value is required in `kpt cfg create-setter`.

Reply:
Value is required to create a setter. what is the use-case for not creating setter with value ? In this scenario, you can have setter with value YOUR_NAMESPACE and mark it as required. 

https://googlecontainertools.github.io/kpt/guides/producer/setters/#marking-a-field-as-required

Reply:
Done - marking it as required